test/functional/wallet_reorgsrestore.py

```diff
@@ -90,7 +90,7 @@ def test_coinbase_automatic_abandon_during_startup(self):
         assert_equal(wallet0.gettransaction(descendant_tx_id)['details'][0]['abandoned'], True)
 
     def test_reorg_handling_during_unclean_shutdown(self):
-        self.log.info("Test that wallet doesn't crash due to a duplicate block disconnection event after an unclean shutdown")
+        self.log.info("Test that wallet transactions are un-abandoned in case of temporarily invalidated blocks and wallet doesn't crash due to a duplicate block disconnection event after an unclean shutdown")
```

Comment:
nit: I haven't seen the format I'm proposing here but just to make it more clear if it makes sense...
```suggestion
        self.log.info(
            "Test that on a duplicate block disconnection event after unclean shutdown:\n"
            "    - wallet transactions are un-abandoned after temporarily invalidated blocks;\n"
            "    - wallet doesn't crash."
)
```

Reply:
I recall spending some time to come up with a log format specifically for this but I was not satisfied with the ones I came up, and thus defaulted to the regular one.
This suggestion looks better to me, I will use it if I end up retouching.